main.py

```diff
@@ -0,0 +1,203 @@
+import json
+import requests
+import urllib
+import time
+from collections import defaultdict
+import email
+import imaplib
+import sys
+from pony import orm
+import quopri
+
+
+db = orm.Database()
+class Chats(db.Entity):
+    chat_id = orm.PrimaryKey(int)
+    state = orm.Required(int)
+    login = orm.Optional(str)
+    passwd = orm.Optional(str)
+db.bind(provider='sqlite', filename=':memory:')
+db.generate_mapping(create_tables=True)
+
+
+TOKEN = "5295625360:AAH5BaLlkwmC4iIRFbusIMSPREhaehbWykk"
```

Comment:
There is a good instrument to work with secret tokens: https://github.com/theskumar/python-dotenv

And a constant string in the middle of the code seems not to be the best solution.